src/components/graph/menus/create-node-menu.js

```diff
@@ -125,8 +139,13 @@ const CreateNodeMenu = ({
         },
         CUT_MODIFICATION_NODE: {
             onRoot: false,
-            action: () => cutNetworkModificationNode(),
-            id: 'cutNetworkModificationNode',
+            action: () =>
+                isAlreadySelectedForCut()
+                    ? cancelCutNetworkModificationNode()
+                    : cutNetworkModificationNode(),
+            id: isAlreadySelectedForCut()
+                ? 'cancelCutNetworkModificationNode'
+                : 'cutNetworkModificationNode',
```

Comment:
I think it would be cleaner to create a new menu entry instead

Reply:
we want a dynamic menu entry here (to avoid 2 entries that could be enable / disable).
So I think this is the easier way to do it